examples/dmd/wave_equation.cpp

```diff
@@ -0,0 +1,630 @@
+ /*
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+//
+//                       libROM MFEM Example: Wave_Equation (adapted from ex23.cpp)
+//
+// Compile with: make wave_equation 
+//
+// =================================================================================
+//
+// Sample runs and results for Time-Windowing DMD:
+//
+// Command 1:
+//  wave_equation -o 4 -tf 5 -nwinsamp 25 
+//
+// Output 1:
+// Relative error of DMD temperature (u) at t_final: 5 is 3.0483662e-05
```

Comment:
The field is not "temperature." Please replace it with either "solution" or "state."

Reply:
Fixed.